README.md

```diff
@@ -1,44 +1,62 @@
 # Paste Prototyping Kit
 
-## Quick deploy
+Use this kit to create realistic prototypes using Paste and validate your designs. Read below for a guide to getting started.
+
+## Getting set up (for designers):
+
+Deploy to Codesandbox to quickly get started on your prototype
 
-Deploy to Codesandbox
 [![Edit paste-prototype-kit in Codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/twilio-labs/paste-prototype-kit)
 
-Deploy to Vercel
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/twilio-labs/paste-prototype-kit/tree/main)
+If you're logged into Codesandbox, click the blue "fork" button to start editing the prototype.
 
-Deploy to Netlify
-[![Deploy with Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/twilio-labs/paste-prototype-kit)
+## Getting set up (for developers):
 
-## Getting Started
+After forking and cloning the repo to your local machine, run
 
-First, run the development server:
+```bash
+npm install
+# or
+yarn install
+```
+then, run the development server
```

Comment:
Is there a reason there are npm and yarn instructions for this and only yarn for testing?

Reply:
nope, good catch!